### PR TITLE
Bump version to 6.3.0 to prepare for release.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Prepare stagenet release.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: collector-backend/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "collector-backend"
-version = "1.1.3"
+version = "1.1.4"
```
Comment (EmilLa1) [inline collector-backend/Cargo.toml:3]: Why is this bumped?

**Review by EmilLa1 — APPROVED** [review]
What about the windows installer?

File: collector-backend/Cargo.toml (hunk 1 above)
Comment (abizjak) [reply collector-backend/Cargo.toml:3]: Because some dependencies were updated to address some security advisories on them.

**Comment by abizjak** [comment]
> What about the windows installer?

> What about the windows installer?

I don't know what you mean, but since this PR https://github.com/Concordium/concordium-node/pull/1043/files was merged (which you approved :P) we don't have to manually change things in the windows installer anymore.

**Pull request merged**